integration-tests/src/test/java/oracle/weblogic/kubernetes/utils/DbUtils.java

```diff
@@ -719,4 +689,331 @@ public static void updateRcuAccessSecret(String secretName, String namespace,
 
   }
 
+  /**
+   * Install Oracle Database Operator.
+   *
+   * @param namespace name of the namespace
+   * @throws IOException when fails to modify operator yaml file
+   */
+  public static void installDBOperator(String namespace) throws IOException {
+    String operatorYamlFile = DOWNLOAD_DIR + "/oracle-database-operator.yaml";
+    String certManager = "https://github.com/jetstack/cert-manager/releases/latest/download/cert-manager.yaml";
+    CommandParams params = new CommandParams().defaults();
+    params.command("kubectl apply -f " + certManager);
+    boolean response = Command.withParams(params).execute();
+    assertTrue(response, "Failed to install cert manager");
+
+    assertDoesNotThrow(() -> TimeUnit.SECONDS.sleep(60));
+
+    String operatorYamlUrl = "https://raw.githubusercontent.com/"
+        + "oracle/oracle-database-operator/" + ORACLE_DB_OPERATOR_RELEASE + "/oracle-database-operator.yaml";
+
+    Files.createDirectories(Paths.get(DOWNLOAD_DIR));
+    Files.deleteIfExists(Paths.get(DOWNLOAD_DIR, "oracle-database-operator.yaml"));
+    params = new CommandParams().defaults();
+    params.command("curl -fL " + operatorYamlUrl + " -o " + operatorYamlFile);
+    response = Command.withParams(params).execute();
+    assertTrue(response, "Failed to download Oracle operator yaml file");
+    replaceStringInFile(Paths.get(operatorYamlFile).toString(),
+        "replicas: 3", "replicas: 1");
+    replaceStringInFile(Paths.get(operatorYamlFile).toString(),
+        "oracle-database-operator-system", namespace);
+
+    params = new CommandParams().defaults();
+    params.command("kubectl apply -f " + operatorYamlFile);
+    response = Command.withParams(params).execute();
+    assertTrue(response, "Failed to install Oracle database operator");
+
+    String dbOpPodName = "oracle-database-operator-controller-manager";
+
+    // wait for the pod to be ready
+    getLogger().info("Wait for the database operator {0} pod to be ready in namespace {1}",
+        dbOpPodName, namespace);
+    testUntil(
+        assertDoesNotThrow(()
+            -> podIsReady(namespace, null, dbOpPodName), "Checking for database pod ready threw exception"),
+        getLogger(), "Waiting for database operator {0} to be ready in namespace {1}", dbOpPodName, namespace);
+
+  }
+
+  /**
+   * Uninstall DB operator.
+   *
+   * @param namespace namespace in which DB is operator running
+   */
+  public static void uninstallDBOperator(String namespace) {
+    String operatorYamlFile = DOWNLOAD_DIR + "/oracle-database-operator.yaml";
+    String certManager = "https://github.com/jetstack/cert-manager/releases/latest/download/cert-manager.yaml";
```

Comment:
Can we move this hard-coded URL to TestConstant class

Reply:
done